### ISSUE TITLE
Documentation includes static list of contributors, not updated

### ISSUE THREAD

**Issue by cpavlina** [ciampix/kicad-doc#75]
All the documentation contains something like this:

    *Contributors*
    
    Jean-Pierre Charras, Fabrizio Tappero.

Quite a few more people have contributed documentation, but nobody remembers  to add names to the list. Perhaps we can either remove the explicit lists in favor of just referring to "the KiCad documentation contributors", or find a way to have that pulled out of the git history automatically?

**Comment by nickoe** [comment]
Yeah, I have been thinking about that before, but I have done nothing about it because I want to hear other peoples opinion. I don't quite have one yet.

**Comment by BrianSidebotham** [comment]
I think we'll want to end up with something like `git log --stat --pretty=format:'%cN'` (or similar, there's a million possibilities here) and then pass it to some CMake foo which can generate a list of contributors, even on a per-document AND per-language basis.

**Comment by Caerbannog** [comment]
Do we need the contributor list in the first place? The git repository is public, we could just link it.

**Comment by nickoe** [comment]
Did we ever reach a consensus or decision about this?

**Comment by heitorPB** [comment]
What about using the output of `git shortlog --summary --numbered` to make a contributors page instead of having the name of only a few people in each page?

**Comment by ciampix** [comment]
Oh nice command! I can't say... you see, I'm biased ... ;-)

**Comment by heitorPB** [comment]
Just notice that my name disappeared from the git history as https://github.com/KiCad/kicad-doc/pull/680 was squash-merged. Maybe this happened some other times as well? Another problem to get the contributors from git is that not every user adds their real name here.

Maybe a `contributors.txt` with everybody alphabetically is better?

**Comment by ciampix** [comment]
So just a `git shortlog --summary |gawk '{print $2}'|sort`  should suffice?

**Comment by heitorPB** [comment]
What about `KiCad Documentation Team` with a link to https://github.com/KiCad/kicad-doc/graphs/contributors ?

**Comment by heitorPB** [comment]
I just noticed that in KiCad -> Help -> About there's a list of documentation writers. There are 6 names in there. There's also a list of devs, artists, translators and packagers. Should we find out a way to keep in sync the list in the software and also in the docs?

**Comment by heitorPB** [comment]
I just found out the project [AllContributors](https://allcontributors.org/) to help in this task.
